scripts/rebranding.ps1

```diff
@@ -32,8 +32,13 @@ Param(
     [int]$buildNumber
 )
 
+$verbose = $false
+if ($PSBoundParameters.ContainsKey('Verbose')) {
+    $verbose = $PsBoundParameters['Verbose']
+}
```

Comment:
why are we not defining it as a non mandatory parameter? (same for the main build script)

Reply:
Because it conflicts with the existing PowerShell `-Verbose` switch which is there by default.